src/gluonts/transform/feature.py

```diff
@@ -30,6 +30,66 @@ def target_transformation_length(
     return target.shape[-1] + (0 if is_train else pred_length)
 
 
+class ImputationStrategy:
+    none = "none"
+    dummy_value = "dummy_value"
+    mean = "mean"
+    median = "median"
+    last_value = "last_value"
+
+    @staticmethod
```

Comment:
We do not want to define static methods.

Reply:
Thanks! Fixed.